### PR TITLE
Changes to move overview toolbar actions to be aligned with the heading, as well as behave better at responsive viewport widths

### PR DESCRIPTION
**Pull request by sg00dwin**
Part of https://jira.coreos.com/browse/CONSOLE-773

screen shot 2018-09-11 at 4 58 15 pm
screen shot 2018-09-11 at 5 34 50 pm

screen shot 2018-09-11 at 5 36 25 pm

### CODE REVIEW

File: frontend/public/components/_overview.scss [hunk 1]
```diff
@@ -1,7 +1,27 @@
...
 
 .overview-toolbar__dropdown {
   display: inline-block;
+  max-width: 100%; // enable text-overflow: ellipsis
+  @media (max-width: $screen-xs-max) {
+    min-width: 100%;
+  }
+}
+
+.overview-toolbar__dropdown .btn-dropdown {
```
Comment (rhamilto) [inline frontend/public/components/_overview.scss]: NIt:  nest under above?

File: frontend/public/components/_overview.scss [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+  max-width: 280px;
+}
+
+.overview-toolbar {
```
Comment (rhamilto) [inline frontend/public/components/_overview.scss]: Nit:  alpha

File: frontend/public/components/overview.jsx [hunk 3]
```diff
@@ -264,13 +264,20 @@ class OverviewDetails extends React.Component {
...
 
     return <div className="co-m-pane">
-      <OverviewToolbar
-        filterValue={filterValue}
-        groupOptions={groupOptions}
-        handleFilterChange={this.handleFilterChange}
-        handleGroupChange={this.handleGroupChange}
-        selectedGroup={selectedGroupLabel}
-      />
+      <div className="co-m-nav-title project-overview__filter-groupby">
```
Comment (rhamilto) [inline frontend/public/components/overview.jsx]: There is no`project-overview` block, so `project-overview__filter-groupby` is out of compliance with BEM guidelines.  I recommend a modifier on `co-m-nav-title` (e.g., `co-m-nav-title--overview`).

File: frontend/public/components/_project-overview.scss [hunk 4]
```diff
@@ -1,3 +1,13 @@
+.project-overview__filter-groupby {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
```
Comment (jhadvig) [inline frontend/public/components/_project-overview.scss]: nit: the styles should be in alphabetic order

**Comment by sg00dwin** [comment]
@rhamilto It's ready for another look

**Comment by spadgett** [comment]
This will probably conflict with #524 :/

cc @TheRealJon

**Comment by TheRealJon** [comment]
@spadgett No worries, I can rebase and fix any conflicts.

File: frontend/public/components/_nav-title.scss [hunk 5]
```diff
@@ -28,3 +28,13 @@
     padding-top: ($grid-gutter-width / 2);
   }
 }
+
+.co-m-nav-title--overview {
```
Comment (rhamilto) [inline frontend/public/components/_nav-title.scss]: Nit:  nest this in the above rule so it's consistent with the other modifier rules.

File: frontend/public/components/_overview.scss [hunk 6]
```diff
@@ -1,7 +1,26 @@
...
+  padding: 0;
+}
+
 .overview-toolbar__label {
-  margin-right: 10px;
+  margin: 0 10px 0 0;
+  vertical-align: middle;
 }
 
 .overview-toolbar__dropdown {
```
Comment (rhamilto) [inline frontend/public/components/_overview.scss]: Nit:  not new, this should be the second rule in the file (alpha)

**Comment by sg00dwin** [comment]
@rhamilto ptal

**Pull request merged**